src/pages/post-outwards/index.js

```diff
@@ -145,49 +213,23 @@ const Postoutwards = () => {
                     values={formik.values}
                     valueField='recordId'
                     displayField={['reference', 'name', 'flName']}
-                    required
```

Comment:
why did you remove the required here if this field is required?

Reply:
not required any more

Reply:
than please remove it from the validationSchema